pkg/api/server/register_pods.go

```diff
@@ -47,7 +47,7 @@ func (s *APIServer) registerPodsHandlers(r *mux.Router) error {
 	//   name: create
 	//   description: attributes for creating a pod
 	//   schema:
-	//     $ref: "#/definitions/PodSpecGenerator"
+	//     $ref: "#/definitions/PodCreateOptions"
```

Comment:
This should stay PodSpecGenerator

Reply:
leftover from previous iteration thanks for catching that